src/languageservice/decompressProvider.ts

```diff
@@ -7,18 +7,59 @@
 
 import {IDecompressProvider, IPackage} from './interfaces';
 import  {ILogger} from '../models/interfaces';
-const decompress = require('decompress');
+import { PlatformInformation } from '../models/platform';
+
+const DecompressZip = require('decompress-zip');
```

Comment:
Any reason not to use yauzl like the rest of our stuff?

Reply:
Not really, I just used the most used libraries